repos/dir.targets

```diff
@@ -334,6 +344,8 @@
       <PrebuiltPackages Include="$(PrebuiltPackagesPath)*.nupkg" />
       <PrebuiltPackages Include="$(TarballPrebuiltPackagesPath)*.nupkg" Condition="'$(TarballPrebuiltPackagesPath)' != ''"/>
 
+      <PrebuiltPackages Include="$(CustomUsageCheckPackageDir)*.nupkg" Condition="'$(CustomUsageCheckPackageDir)' != ''" />
```

Comment:
I don't get what's happening with CustomUsageCheckPackageDir here.

Reply:
Ah, yeah, this is kind of out of nowhere here. I'll add a comment. The idea is it's used by `FindSourceBuiltUsages` to do a "custom usage check".

Reply:
I see that, but why is it included in PrebuiltPackages?

Reply:
Heh, good point... I was being lazy and using the existing target without much modification just out of to convenience. I'll reorganize this to make sense.

I'll also try to make it run immediately after it detects the problem and focus the report the specific problematic package(s). If you just do what the message suggests and run this after the build completes, there's too much junk to practically sift through.